### PR TITLE
fix: typescript imports in the ts-plugin

### PR DESCRIPTION
**Pull request by Princesseuh**
## Changes

After https://github.com/withastro/language-tools/pull/532, we had some random TypeScript imports in the code. We can't have that, we should always use the TypeScript that's passed down to us from the `src/index.ts`, because we don't necessarily have TypeScript in the user's project.

## Testing

Tested manually, can't really test this in our monorepo, we always have access to TypeScript, that's why it wasn't caught previously.

## Docs

N/A

### CODE REVIEW

**Review by MoustaphaDev — APPROVED** [review]
I'm not familiar with this repo but the code looks right to me!

File: packages/ts-plugin/src/language-service/diagnostics.ts [hunk 1]
```diff
@@ -1,13 +1,15 @@
-import ts from 'typescript/lib/tsserverlibrary';
+import type ts from 'typescript/lib/tsserverlibrary';
 import { isAstroFilePath } from '../utils.js';
 
+type _ts = typeof ts;
```
Comment (jasikpark) [inline packages/ts-plugin/src/language-service/diagnostics.ts:4]: why this rename? what does it signify?

Comment (jasikpark) [reply packages/ts-plugin/src/language-service/diagnostics.ts:4]: ohh it's a type instead of a variable

**Review by jasikpark — COMMENTED** [review]
lgtm?

**Review by matthewp — APPROVED** [review]

**Pull request merged**